docs/robot_advanced.rst

```diff
@@ -117,32 +117,68 @@ With the ``PageObjects`` library, you can define classes that represent page obj
 The ``pageobject`` Decorator
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Page objects are normal Python classes that use the ``pageobject`` decorator provided by CumulusCI. Unlike traditional Robot Framework keyword libraries, you can define multiple sets of keywords in a single file.
+Page objects are normal Python classes that use the ``pageobject`` decorator provided by CumulusCI. Unlike traditional Robot Framework keyword libraries, you can easily define and use keywords in multiple classes within a single file.
```

Comment:
I recommend removing "easily." For one thing, how easily? Adverbs make things less specific. For another, for that one reader out there who doesn't find it easy, we're kind of making them feel bad :D

Reply:
Thanks for the suggestion!